dotcom-rendering/src/components/ArticleMeta.apps.tsx

```diff
@@ -0,0 +1,351 @@
+	children: React.ReactNode;
+	isPictureContent: boolean;
+}) => (
+	<div
+		data-print-layout="hide"
+		css={css`
+			grid-area: comment-count;
+			border-left: 1px solid ${themePalette('--article-border')};
+
+			${until.desktop} {
+				margin-top: ${isPictureContent ? '-6px' : '-2px'};
+				padding-top: ${isPictureContent ? '6px' : '2px'};
+				padding-left: 10px;
+			}
+		`}
+	>
+		{children}
+	</div>
+);
+
+const MetaGridDateline = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			grid-area: dateline;
+		`}
+	>
+		{children}
+	</div>
+);
+
+const metaPadding = (format: ArticleFormat) => {
+	if (
+		format.design === ArticleDesign.LiveBlog ||
+		format.design === ArticleDesign.DeadBlog
+	) {
+		return css`
+			${between.tablet.and.leftCol} {
+				order: 3;
+			}
+
+			padding-top: 2px;
+		`;
+	}
+
+	return css`
+		${between.tablet.and.leftCol} {
+			order: 3;
+		}
+
+		${until.mobileLandscape} {
+			padding-left: 10px;
+			padding-right: 10px;
+		}
+
+		${from.mobileLandscape} {
+			padding-left: 20px;
+			padding-right: 20px;
+		}
+
+		${from.phablet} {
+			padding-left: 0px;
+			padding-right: 0px;
+		}
+
+		padding-top: 2px;
+	`;
+};
+
+const stretchLines = (isPictureContent: boolean) => css`
+	display: block;
+
+	${!isPictureContent && until.mobileLandscape} {
+		margin-left: -20px;
+		margin-right: -20px;
+		width: auto;
+	}
+
+	${!isPictureContent && until.phablet} {
+		margin-left: -10px;
+		margin-right: -10px;
+		width: auto;
+	}
+`;
+
+export const ArticleMetaApps = ({
+	branding,
+	format,
+	pageId,
+	webTitle,
+	byline,
+	tags,
+	primaryDateline,
+	secondaryDateline,
+	discussionApiUrl,
+	shortUrlId,
+	isCommentable,
+	ajaxUrl,
+	messageUs,
+}: Props) => {
+	const soleContributor = getSoleContributor(tags, byline);
+	const authorName = soleContributor?.title ?? 'Author Image';
+
+	const avatarUrl = shouldShowAvatar(format)
+		? soleContributor?.bylineLargeImageUrl
+		: undefined;
+	const isInteractive = format.design === ArticleDesign.Interactive;
+
+	const isPictureContent = format.design === ArticleDesign.Picture;
+	// const isLiveBlog = format.design === ArticleDesign.LiveBlog;
+
+	const isCommentLayout = format.design === ArticleDesign.Comment;
+	const shouldShowFollowButtonsOnCommentLayout =
+		isCommentLayout && !!byline && soleContributor !== undefined;
+
+	return (
+		<div
+			className={
+				isInteractive ? interactiveLegacyClasses.metaContainer : ''
+			}
+			css={metaContainerMargins(format)}
+		>
+			<div css={[metaPadding(format)]}>
+				<div>
+					{branding && (
+						<Island priority="feature" defer={{ until: 'visible' }}>
+							<Branding branding={branding} format={format} />
+						</Island>
+					)}
+					{format.theme === ArticleSpecial.Labs ? (
+						<div>
+							<StraightLines
+								cssOverrides={stretchLines(isPictureContent)}
+								count={1}
+								color={border.primary}
+							/>
+							<div
+								css={css`
+									height: ${space[1]}px;
+								`}
+							/>
```

Comment:
As much as possible we should avoid `cssOverrides` for Source components. For a single straight line, I am not sure we need to be using this component. Is this ported from the existing design?

I also think the colour should be `themePalette('--article-border')`.

Reply:
I'm just deleting this as it's duplicated in the StandardLayout as GuardianLabsLines. 

Labs looks weird to me but I haven't seen any designs for it and this is how the GuardianLabsLines component is rendering:
<img width="549" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/32312712/7cbd7cb2-7d72-4022-9bac-9e1d406ba570">